recipes/vispy/ctypes_fontconfig.diff

```diff
@@ -0,0 +1,20 @@
+diff --git a/vispy/ext/fontconfig.py b/vispy/ext/fontconfig.py
+index ff24662..8980629 100644
+--- a/vispy/ext/fontconfig.py
++++ b/vispy/ext/fontconfig.py
+@@ -1,5 +1,6 @@
+ # -*- coding: utf-8 -*-
+ 
++import os, sys
+ import warnings
+ from ctypes import (util, cdll, c_void_p, c_char_p, c_double, c_int, c_bool,
+                     Union, Structure, byref, POINTER)
+@@ -7,7 +8,7 @@ from ..util.wrappers import run_subprocess
+ 
+ # Some code adapted from Pyglet
+ 
+-fc = util.find_library('fontconfig')
++fc = util.find_library(os.path.join(sys.prefix, 'lib', 'fontconfig'))
```

Comment:
I ma not sure that is what you want.  Note that `find_library` returns the library name, like

```python
In [1]: from ctypes import util
In [2]: util.find_library('fontconfig')
Out[2]: 'libfontconfig.so.1'
```

Because the build is failing to find the `libfontconfig.so` at build time what you have to do is to "hard-code" the conda directory of that library. like:

```python
fc = os.path.join(sys.prefix, 'lib', 'libfontconfig.so')
```

and let `cdll.LoadLibrary(fc)` do the rest.